packages/fetch/README.md

```diff
@@ -27,3 +27,5 @@ You can find below some samples
 - [nx fastify react](https://github.com/anymaniax/orval/tree/master/samples/nx-fastify-react)
 - [angular app](https://github.com/anymaniax/orval/tree/master/samples/angular-app)
 - [hono](https://github.com/anymaniax/orval/tree/master/samples/hono)
+- [next app with fetch](https://github.com/anymaniax/orval/tree/master/samples/next-app-with-fetch)
```

Comment:
it looks like you copied this line twice?

Reply:
oops, that's right.

Reply:
fixed by bellow:

https://github.com/anymaniax/orval/pull/1407/commits/939975405b3a6c6eb4dcf53f9d9f0e135a5b54e4